controllers/users/chatSearch.js

```diff
@@ -38,19 +40,20 @@ const chatSearch = async (req, res) => {
     let additionalQuery = followed?.length > 0 ? `AND A.user_id NOT IN (:followedIds)` : '';
 
     let moreUserSearchQuery = `
-  SELECT B.user_id_follower, B.user_id_followed, * 
-  FROM users A 
-  LEFT JOIN user_follow_user B
-  ON B.user_id_followed = :userId AND B.user_id_follower = A.user_id
-  WHERE 
-      A.username ILIKE :query AND 
-      A.user_id != :userId AND
-      A.verified_status != 'UNVERIFIED' AND
-      A.is_anonymous = false
-      ${additionalQuery}
-  ORDER BY B.user_id_followed
-  LIMIT :limit
-  OFFSET :offset`;
+      SELECT B.user_id_follower, B.user_id_followed, * 
+      FROM users A 
+      LEFT JOIN user_follow_user B
+      ON B.user_id_followed = :userId AND B.user_id_follower = A.user_id
+      WHERE 
+          A.username ILIKE :query AND 
+          A.user_id != :userId AND
+          A.verified_status != 'UNVERIFIED' AND
+          A.is_anonymous = false AND
+          A.user_id != :adminUserId
+          ${additionalQuery}
+      ORDER BY B.user_id_followed
+      LIMIT :limit
+      OFFSET :offset`;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Same issues as above with the SQL query and the use of environment variables.
```diff
- SELECT B.user_id_follower, B.user_id_followed, * 
+ SELECT B.user_id_follower, B.user_id_followed, A.username, A.user_id, A.verified_status, A.is_anonymous

- adminUserId: process.env.BETTER_ADMIN_ID
+ adminUserId
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->